ts-apps/decision-logic-visualizer/src/lib/l4-connection.ts

```diff
@@ -18,16 +18,24 @@ export class L4Connection {
   // TODO: Think about how we can remove the need for consumers of L4Connection to supply a `verDocId`
```

Comment:
Basically we need a way for the instance of the LadderFlow component to trigger an update of the component from the parent